application/helpers/previewer_helper.php

```diff
@@ -30,6 +30,9 @@ function clean_title($title, $retval = 'title')
 	return $prefix;
 }
 
+// Note: The create_full_title function is also used by the WordPress theme
+// (also see comment before format_authors).
+
```

Comment:
Shouldn't we not leave an empty line here?

Reply:
I think there are enough extra blank lines in this project to _(insert witty comment here)_. Maybe it is contagious?

I just pushed a change to remove these lines and change the "Note" to "WARNING".